src/Core/src/Handlers/Editor/EditorHandler.iOS.cs

```diff
@@ -138,8 +129,23 @@ public static void MapFormatting(IEditorHandler handler, IEditor editor)
 			handler.PlatformView?.UpdateCharacterSpacing(editor);
 		}
 
-		public static void MapIsEnabled(IEditorHandler handler, IEditor editor) =>
+		public static void MapIsEnabled(IEditorHandler handler, IEditor editor)
+		{
+#if !MACCATALYST
+			if (editor.IsEnabled)
+			{
+				var _accessoryView = new MauiDoneAccessoryView();
```

Comment:
Can this check if the accessory view has already been set and then just not re-instantiate it?